src-site/components/map.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Map
+---
+
+<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css" integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY=" crossorigin=""/>
+<script src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js" integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo=" crossorigin=""></script>
+
+Use the Map component to show the user geographic information. For example:
+
+- An electoral boundary (e.g: their constituency/ward)
+- The location of a place (e.g: their assigned polling station)
+- The route to a place (e.g: from their house to their assigned polling station)
+
+We use <a href="https://leafletjs.com/" target="_blank">Leaflet.js</a> as our mapping library.
+
+{% ds-example %}
+<div id="area_map" class="ds-map"></div>
+{% endds-example %}
+
+<script>
+    // light mode
+    L.tileLayer("https://tile.openstreetmap.org/{z}/{x}/{y}.png", {
+        maxZoom: 19,
+        attribution:
+            '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+    }).addTo(L.map("area_map").setView([51.505, -0.09], 13));
+
+    // dark mode
+    L.tileLayer("https://tile.openstreetmap.org/{z}/{x}/{y}.png", {
+        maxZoom: 19,
+        attribution:
+            '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a>'
+    }).addTo(L.map("dark_area_map").setView([51.505, -0.09], 13));
+</script>
+
+```html
+Here’s a demo:
+
+<div id="area_map" class="ds-map"></div>
```

Comment:
For the example here I haven't included any of the required leaflet CSS/JS to make this actually work. Just the HTML container div. I think how we pull in leaflet and write the JS to render the map is probably out of scope for the design system?

Reply:
I agree, this is good enough to know how to use it without needing to pull leaflet into this project.